generate_imgui_bindings.jai

```diff
@@ -16,8 +16,20 @@ generate_imgui_bindings :: () {
     opts: Generate_Bindings_Options;
     {
         using opts;
-        array_add(*libpaths,      "./win/dll");
         array_add(*libnames,      "imgui");
+        #if OS == .WINDOWS {
+            array_add(*libpaths,      "./win/dll");
+        }
+        #if OS == .LINUX {
+            array_add(*libpaths,      "./linux");
+            array_add(*include_paths, "/usr/include/c++/10");
+            array_add(*include_paths, "/usr/include/x86_64-linux-gnu/c++/10");
+            array_add(*include_paths, "/usr/include/c++/10/backward");
+            array_add(*include_paths, "/usr/local/include");
+            array_add(*include_paths, "/usr/lib/llvm-12/lib/clang/12.0.0/include");
+            array_add(*include_paths, "/usr/include/x86_64-linux-gnu");
+            array_add(*include_paths, "/usr/include");

```

Comment:
One question about these include_path additions--do you know if it is maybe more correct to say something like

```
for split(getenv("INCLUDE"), ":") array_add(*include_paths, it);
```

instead? Or is that actually more brittle and less maintainable? Not a Linux expert in any way here 😅